### PR TITLE
Updates to the creating your first artboard topic

### LINKED ISSUE #315: Update getting started to use text styles not symbols
Update getting started to use text styles not symbols

### LINKED ISSUE #319: Update minimist version to 1.2.5
Update minimist version to 1.2.5

### PR DESCRIPTION
**Pull request by StefanIvanov**
Closes #315 
Closes #319

### CODE REVIEW

**Comment by StefanIvanov** [comment]
@randriova will do the localization within this PR and then we will merge it

**Comment by StefanIvanov** [comment]
Thanks @randriova we are ready to merge this one once @ig-robstoffers or @pmoleri approve the changes.

**Comment by StefanIvanov** [comment]
@ig-robstoffers can you test this once you are finished with the release

File: en/creating-an-artboard.md [hunk 1]
```diff
@@ -40,30 +40,30 @@ If the [Titillium Web Font][1] isn't installed on your computer, please do it be
...
-If you added the Libraries successfully, they will show up in the **Symbols** section of the **Insert** menu in Sketch.
+  <div class="divider--half"></div>
+  <img class="responsive-img" src="images/creating_an_artboard_2.png" />
+  <div class="divider--half"></div>
+  <div class="divider--half"></div>
+  <div class="divider--half"></div>
+  <div class="divider--half"></div>
+  <div class="divider--half"></div>
+
+You may have noticed that in the image above the libraries also appear to be present in my Sketch cloud, and as an alternative, you can use this mechanism in Sketch if your are running version 57 or later. The set up is very easy, you only need to open Styling, Components, and Patterns one after another and save each file in your Sketch cloud. A detailed explanation of how to set Indigo.Design as cloud libraries can be found in [this Medium post](https://medium.com/ignite-ui/using-indigo-design-with-sketch-cloud-libraries-70533bca00f8). Once you added the Libraries successfully, they will show up in the **Symbols** section of the **Insert** menu in Sketch.
```
Comment (Aleksandyr) [inline en/creating-an-artboard.md:59]:
```suggestion
You may have noticed that in the image above the libraries also appear to be present in my Sketch cloud, and as an alternative, you can use this mechanism in Sketch if you are running version 57 or later. The set up is very easy, you only need to open Styling, Components, and Patterns one after another and save each file in your Sketch cloud. A detailed explanation of how to set Indigo.Design as cloud libraries can be found in [this Medium post](https://medium.com/ignite-ui/using-indigo-design-with-sketch-cloud-libraries-70533bca00f8). Once you added the Libraries successfully, they will show up in the **Symbols** section of the **Insert** menu in Sketch.
```

**Pull request merged**